pkg/util/validation/limits.go

```diff
@@ -101,6 +101,8 @@ type Limits struct {
 	MaxGlobalMetadataPerMetric          int `yaml:"max_global_metadata_per_metric" json:"max_global_metadata_per_metric"`
 	// Exemplars
 	MaxGlobalExemplarsPerUser int `yaml:"max_global_exemplars_per_user" json:"max_global_exemplars_per_user" category:"experimental"`
+	// Native histograms
+	AcceptNativeHistograms bool `yaml:"accept_native_histograms" json:"accept_native_histograms" category:"experimental"`
```

Comment:
```suggestion
	IngestHistogramSamples bool `yaml:"ingest_histogram_samples" json:"ingest_histogram_samples" category:"experimental"`
```

Reply:
Another suggestion from p: `histogram-ingestion-enabled`

Reply:
> Another suggestion from p: histogram-ingestion-enabled

👍

Reply:
hmm, the one problem I have that it doesn't line up with Prometheus and doesn't specify "native histograms":
see --enabled-feature=[native-histograms]https://prometheus.io/docs/prometheus/latest/feature_flags/#native-histograms and [send_native_histograms](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write)

Thus it could be confusing at first glance for the end-user. Imagine looking at a diff.

So how about `native-histograms-ingestion-enabled` ?

Reply:
>So how about native-histograms-ingestion-enabled ?

LGTM, thanks

Reply:
done